plugins/imfile/imfile.c

```diff
@@ -316,7 +317,8 @@ static struct cnfparamdescr inppdescr[] = {
 	{ "statefile", eCmdHdlrString, CNFPARAM_DEPRECATED },
 	{ "readtimeout", eCmdHdlrPositiveInt, 0 },
 	{ "freshstarttail", eCmdHdlrBinary, 0},
-	{ "filenotfounderror", eCmdHdlrBinary, 0}
+	{ "filenotfounderror", eCmdHdlrBinary, 0},
+	{ "needparse", eCmdHdlrBinary, 0}
```

Comment:
this looks like a feature change, not a port.

Reply:
Sorry, You are right. I forgot to remove it ad treat it as a feature. Do want me to pull it as a new PR ?

Reply:
just let's document what it does and why it is nedded. I think I already grasped it and if we can agree on the usefulness, no need to craft a dedicated PR for this one.